client/src/main/java/org/apache/celeborn/client/write/PushState.java

```diff
@@ -88,4 +96,20 @@ public boolean addBatchData(String addressPair, PartitionLocation loc, int batch
   public DataBatches takeDataBatches(String addressPair) {
     return batchesMap.remove(addressPair);
   }
+
+  public void addFlightBatches(int batchId, PartitionLocation loc) {
```

Comment:
I think it's better to pass hostAndPushPort as parameter instead of loc

Reply:
hello, @waitinfuture . Execute me, could tell me why put a string object is better than an object reference?